pages/admin/invite/userRequest.tsx

```diff
@@ -0,0 +1,111 @@
+import { useEffect, useState } from "react";
```

Comment:
userRequests should not be a page at all. the user requests table is supposed to be a component within in the **same** page as the /admin/invite/index.tsx file. please reference the figma wireframes. again, this should not be its own page

Reply:
reference this page for what the /admin/invite page should look like. we want this page to consist of multiple components including ethan's create new user component and your user requests table at the bottom. https://www.figma.com/file/OXSYtCJxra1NzMT7I58HVx/OGSC?node-id=580%3A661